tools/debian_buildenv.sh

```diff
@@ -84,5 +84,9 @@ case "$COMMAND" in
             protobuf-compiler \
             qt5keychain-dev \
             qtdeclarative5-dev \
+            qml-module-qtquick-controls \
```

Comment:
On Fedora it's just qt5-qtquickcontrols + qt5-qtquickcontrols2.

Reply:
Weird, I didn't install any additional dependencies on Arch. Just qt5-declarative.

Reply:
Both qt5-qtquickcontrols and qt5-qtquickcontrols2 are installed on my system. Maybe they are not needed? I need to check this with a mock build sometime.

Reply:
At least they are not required for qt5-declarative. If we need them we have to install them explicitly.